timemachine/cpp/src/rotations.cu

```diff
@@ -0,0 +1,37 @@
+#include "device_buffer.hpp"
+#include "gpu_utils.cuh"
+#include "kernels/k_rotations.cuh"
+#include "kernels/kernel_utils.cuh"
+#include "math_utils.cuh"
+#include "rotations.hpp"
+
+namespace timemachine {
+
+// rotate_coordinates_host is used to test the rotation kernel
+template <typename RealType>
+void rotate_coordinates_host(
```

Comment:
I didn't want to put kernel code into the `wrap_kernals.cpp` file, so I created this function, not sure if there is a preference of where to put this sort of code.